src/Parser/Warlock/Affliction/Modules/Features/Abilities.js

```diff
@@ -8,135 +8,119 @@ class Abilities extends CoreAbilities {
-        cooldown: 120,
-        enabled: combatant.hasTalent(SPELLS.SOUL_HARVEST_TALENT.id),
+        spell: SPELLS.PHANTOM_SINGULARITY_TALENT,
+        category: Abilities.SPELL_CATEGORIES.ROTATIONAL_AOE,
+        cooldown: 45,
+        enabled: combatant.hasTalent(SPELLS.PHANTOM_SINGULARITY_TALENT.id),
+        isOnGCD: true,
         castEfficiency: {
           suggestion: true,
-          recommendedEfficiency: 0.9,
         },
       },
       {
-        spell: SPELLS.SUMMON_DOOMGUARD_UNTALENTED,
-        category: Abilities.SPELL_CATEGORIES.COOLDOWNS,
-        cooldown: 180,
-        enabled: !combatant.hasTalent(SPELLS.GRIMOIRE_OF_SUPREMACY_TALENT.id),
+        spell: SPELLS.SEED_OF_CORRUPTION_DEBUFF,
+        category: Abilities.SPELL_CATEGORIES.ROTATIONAL_AOE,
         isOnGCD: true,
+      },
+      {
+        spell: SPELLS.VILE_TAINT_TALENT,
+        category: Abilities.SPELL_CATEGORIES.ROTATIONAL_AOE,
+        isOnGCD: true,
+        enabled: combatant.hasTalent(SPELLS.VILE_TAINT_TALENT.id),
+        cooldown: 20,
         castEfficiency: {
           suggestion: true,
-          recommendedEfficiency: 0.95,
+          recommendedEfficiency: 0.8,
         },
       },
+
+      // Cooldowns
       {
-        spell: SPELLS.SUMMON_INFERNAL_UNTALENTED,
+        spell: SPELLS.SUMMON_DARKGLARE,
         category: Abilities.SPELL_CATEGORIES.COOLDOWNS,
         cooldown: 180,
-        enabled: !combatant.hasTalent(SPELLS.GRIMOIRE_OF_SUPREMACY_TALENT.id),
         isOnGCD: true,
         castEfficiency: {
           suggestion: true,
-          recommendedEfficiency: 0.95,
+          recommendedEfficiency: 0.9,
         },
       },
       {
-        spell: [
-          SPELLS.GRIMOIRE_OF_SERVICE_TALENT,
-          SPELLS.GRIMOIRE_IMP,
-          SPELLS.GRIMOIRE_SUCCUBUS,
-          SPELLS.GRIMOIRE_FELHUNTER,
-          SPELLS.GRIMOIRE_VOIDWALKER,
-        ],
+        spell: SPELLS.DARK_SOUL_MISERY_TALENT,
         category: Abilities.SPELL_CATEGORIES.COOLDOWNS,
-        cooldown: 90,
-        enabled: combatant.hasTalent(SPELLS.GRIMOIRE_OF_SERVICE_TALENT.id),
-        isOnGCD: true,
+        cooldown: 120,
+        enabled: combatant.hasTalent(SPELLS.DARK_SOUL_MISERY_TALENT.id),
         castEfficiency: {
           suggestion: true,
-          recommendedEfficiency: 0.90,
-          averageIssueEfficiency: 0.80,
-          majorIssueEfficiency: 0.70,
+          recommendedEfficiency: 0.9,
         },
       },
 
-      // Utility
-      {
-        spell: SPELLS.BURNING_RUSH_TALENT,
-        category: Abilities.SPELL_CATEGORIES.UTILITY,
-        enabled: combatant.hasTalent(SPELLS.BURNING_RUSH_TALENT.id),
-        isOnGCD: true,
-      },
+      // Defensive
       {
         spell: SPELLS.UNENDING_RESOLVE,
         buffSpellId: SPELLS.UNENDING_RESOLVE.id,
         category: Abilities.SPELL_CATEGORIES.DEFENSIVE,
         cooldown: 180,
+        isOnGCD: false,
         castEfficiency: {
           suggestion: false,
```

Comment:
This should be true if you want it to show up as a suggestion (which I assume is the case since you have importance set below). It's also the case for a lot of the other defensive or utility spells.

Reply:
I meant this to be like "you should use your defensives at least once" (to help healers during fights), but I probably messed it up. Same goes for Demonic Circle - you should use it probably if you have it talented, but it's very situational (so not cast on cooldown). I'll turn the suggestion off and perhaps make a module to track that instead.

Reply:
all you need to do is set `suggestion` to true and it will work like you want it to.